Mobile.BuildTools/Tasks/ScssProcessorTask.cs

```diff
@@ -83,7 +71,7 @@ private IEnumerable<TaskItem> ProcessFiles(IEnumerable<string> inputFiles)
                 var css = result.Css;
                 Logger.LogMessage(css);
 
-                if (!string.IsNullOrWhiteSpace(MinimizeCSS) && bool.TryParse(MinimizeCSS, out bool b) && b)
+                if (string.IsNullOrWhiteSpace(DebugOutput) || (bool.TryParse(DebugOutput, out bool b) && !b))
```

Comment:
I would instead just rely on the built-in mechanism in MSBuild to determine verbosity. Just invoke LogMessage with an importance of `Low`, and the developer can decide the verbosity level they want in their output just by passing `/v:normal` (the default) or `/v:detailed` (or `diagnostic`) and get the low-priority messages too.

Reply:
in this specific case you're totally correct that I could default to the MSBuild verbosity. Since the other tasks contain sensitive information I've been forcing an explicit opt in if you want unshielded outputs.